### PR TITLE
Desktop website media queries

### PR DESCRIPTION
**Pull request by anas599**
Finishing desktop website and setting media quarries to achieve optimal user experience when using desktop computer
And then some transitions and animation on hover or other effects

### CODE REVIEW

**Comment by huseyinbiyik** [comment]
Hi @anas599,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

- You have submitted a project, that has no .gitignore file. Kindly add the .gitignore file delete test.md file and add node_modules.
- Please remove the logo on the top of your Readme file. If you have your own logo, you can use it.
- Please replace the "Hello Microverse README file" with your project name. And same for the rest, you need to replace placeholders with your custom content. These changes are crucial to have a professional-looking readme. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -239,13 +239,15 @@ <h1>Skills</h1>
                 </form>
             </section>
         </section>
+        <footer>
```
Comment (tufoinnkuo10) [inline index.html:242]: - [x] nice work. for it to look more like the Figma design, Please add the extra space under the email in the contact section 
- [x] kindly reduce the text area, and please respect the Figma margins for the `fullname, email and message`
- [x] kindly reduce the email font size so it matches the Figma design
yours
Screenshot (641)_LI
figma
Screenshot (642)

File: style.css [hunk 2]
```diff
@@ -155,24 +160,34 @@ body {
...
   inline-size: 35%;
+  white-space: nowrap;
+  padding-left: 5px;
+  padding-right: 5px;
 }
 
-.l2 {
+/* .l2 {
   background-color: #ebf0ee;
   list-style: none;
```
Comment (tufoinnkuo10) [inline style.css:192]: - [x] kindly remove commented code to keep your work clean. nice work

File: index.html (hunk 1 above)
Comment (tufoinnkuo10) [inline index.html:242]: - [x] kindly use the correct different background image of the about me section for the mobile and desktop version.
your mobile
Screenshot (645)_LI
figma mobile
Screenshot (646)
your desktop
Screenshot (647)
figma desktop
Screenshot (648)

**Comment by anas599** [comment]
Adding additional CSS attribute

File: index.html [hunk 3]
```diff
@@ -239,13 +239,15 @@ <h1>Skills</h1>
...
         <ul class="social_form">
             <li><a href="#" class="fa fa-facebook-square"></a></li>
             <li><a href="#" class="fa fa-twitter-square"></a></li>
             <li><a href="#" class="fa fa-github-square"></a></li>
             <li><a href="#" class="fa fa-instagram"></a></li>
         </ul>
         <h2 class="eaddress">anas599@yahoo.com</h2>
+        </footer>
```
Comment (tufoinnkuo10) [inline index.html:250]: - [x] nice work with the desktop version, kindly add the background image to the contact section of the desktop version so it matches the Figma design
- [x] add padding to the input placeholders as well so as to follow the Figma design
yours
Screenshot (649)

figma design
Screenshot (650)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @anas599 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ Media query well used
✅ Desktop design well implemented

Required Changes ♻️
Check the comments under the review
tufoin
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**